aspnetcore/grpc/grpcweb.md

```diff
@@ -51,7 +51,7 @@ Alternatively, the gRPC-Web middleware can be configured so that all services su
 :::code language="csharp" source="~/grpc/grpcweb/sample/8.x/GrpcGreeter/Program.cs" id="snippet_WebEnableAllServices" highlight="9":::
 
 > [!NOTE]
-> There is a known issue that causes gRPC-Web to fail when [hosted by HTTP.sys](xref:fundamentals/servers/httpsys) in .NET Core 3.x.
+> There is a known issue that causes gRPC-Web to fail when [hosted by HTTP.sys](xref:fundamentals/servers/httpsys) in .NET 3.x.
```

Comment:
This should still say .NET Core 3.x. Fix this instance and all other references to .NET Core 3.x.